Workbooks/Traffic Analytics/Unified/ApplicationPorts.workbook

```diff
@@ -0,0 +1,955 @@
+          },
+          {
+            "id": "553b6a62-5413-4f6f-b50f-bd2db686aa33",
+            "version": "KqlParameterItem/1.0",
+            "name": "portRange",
+            "label": "Ports range",
+            "type": 2,
+            "description": "Select one or more ports range",
+            "isRequired": true,
+            "multiSelect": true,
+            "quote": "",
+            "delimiter": ",",
+            "query": "datatable (id: int)\r\n[0,\r\n1,\r\n2,\r\n3,\r\n4,\r\n5,\r\n6,\r\n7,\r\n8,\r\n9,\r\n10,\r\n11,\r\n12,\r\n13,\r\n14,\r\n15,\r\n16,\r\n17,\r\n18,\r\n19,\r\n20,\r\n21,\r\n22,\r\n23,\r\n24,\r\n25,\r\n26,\r\n27,\r\n28,\r\n29,\r\n30,\r\n31,\r\n32,\r\n33,\r\n34,\r\n35,\r\n36,\r\n37,\r\n38,\r\n39,\r\n40,\r\n41,\r\n42,\r\n43,\r\n44,\r\n45,\r\n46,\r\n47,\r\n48,\r\n49,\r\n50,\r\n51,\r\n52,\r\n53,\r\n54,\r\n55,\r\n56,\r\n57,\r\n58,\r\n59,\r\n60,\r\n61,\r\n62,\r\n63,\r\n64,\r\n65]\r\n| project value = strcat(tostring(id*1000), '-', tostring(min_of(65535, id*1000+999))), label = strcat(tostring(id*1000), '-', tostring(min_of(65535, id*1000+999))), selected = 1",
+            "crossComponentResources": [
+              "{workspace}"
+            ],
+            "typeSettings": {
+              "additionalResourceOptions": [
+                "value::all"
+              ],
+              "selectAllValue": "*",
+              "showDefault": false
+            },
+            "timeContext": {
+              "durationMs": 1800000
+            },
+            "defaultValue": "value::all",
+            "queryType": 0,
+            "resourceType": "microsoft.operationalinsights/workspaces"
+          },
+          {
+            "id": "84e0ab54-be7e-454d-b382-624f4c18d5b9",
+            "version": "KqlParameterItem/1.0",
+            "name": "isNSGFlowSchemaPresent",
+            "type": 1,
+            "description": "Schema Check",
+            "isRequired": true,
+            "query": "let SchemaTable = view () { print isPresent = 'false' };\r\nunion isfuzzy = true SchemaTable, (AzureNetworkAnalytics_CL | take 1 | project isPresent = iff(iscolumnexists('AllowedInFlows_d'), 'true', 'false')) \r\n| top 1 by isPresent desc",
+            "crossComponentResources": [
+              "{workspace}"
+            ],
+            "isHiddenWhenLocked": true,
+            "defaultValue": "value::all",
+            "queryType": 0,
+            "resourceType": "microsoft.operationalinsights/workspaces"
+          },
+          {
+            "id": "8ba132ac-8838-476c-8eb2-ed42aa2aa983",
+            "version": "KqlParameterItem/1.0",
+            "name": "isVNetTopologySchemaPresent",
+            "type": 1,
+            "description": "Schema Check",
+            "isRequired": true,
+            "query": "let SchemaTable = view () { print isPresent = 'false' };\r\nunion isfuzzy = true SchemaTable, (NTATopologyDetails | getschema | summarize c=count() | project isPresent = iff(c > 0, 'true', 'false')) \r\n| top 1 by isPresent desc",
```

Comment:
why c>0 ?
c is the total number for fields in the schema. Should we not check some finite number indication a minimum number of fields present?
Also, in case we stick to using c>0, then we can avoid summarizing and just use "take 1" or "limit 1". That is faster than summarize.